server/data/scanner.js

```diff
@@ -112,18 +114,27 @@ scanner = () => {
 
           // If the 2 values aren't equal then the lines crossed, If they are both true or both false the lines did not cross
           if (!x === y) {
+            // this is the only way I could get the timing to work
+            // refactor this at some point to increase reliability
+            let test = Date.now() + 5000
+            let z = 0
+            do {
+              z = z + 1;
+            }
+            while (Date.now() < test)
```

Comment:
We might want to do some research into getFullYear  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getFullYear. It has according to local time so it may work better. Just a suggestion doesn't have to be changed right this second.